coderd/database/migrations/000258_add_otp_reset_to_user.up.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE users ADD COLUMN hashed_one_time_passcode bytea;
```

Comment:
Nit: I feel the `hashed_` prefix is unnecessary.
Also I think we could abbreviate these to `otp` for brevity, and you have helpfully added comments explaining what the fields are.

Reply:
We already have the prefix for `hashed_password` so it does maintain consistency

Reply:
The decision to prefix with `hashed_` was that the password column also has that prefix. I'm fine with removing it though.

Reply:
Consistency is good, I forgot about `hashed_password`, but I do still like `otp` as an abbreviation though (but don't feel strongly about it).

Reply:
If we use `otp` then we'll need to add some [_aguisíní_](https://www.focloir.ie/en/dictionary/ei/aguis%C3%ADn) to `sqlc.yaml` to generate `HashedOTP` and `OTPExpiresAt` instead of `HashedOtp` and `OtpExpiresAt`.